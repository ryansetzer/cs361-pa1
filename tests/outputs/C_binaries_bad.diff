--- outputs/C_binaries_bad.txt	2022-09-25 14:55:16.758657842 -0400
+++ expected/C_binaries_bad-2.txt	2022-09-11 14:59:21.000000000 -0400
@@ -0,0 +1,4 @@
+$ ./bin/ls -l
+$ ./bin/head -c 5 Makefile
+$ quit
+
