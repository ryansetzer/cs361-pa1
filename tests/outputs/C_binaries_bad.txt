$ ./bin/ls -l
$ ./bin/head -c 5 Makefile
$ quit

