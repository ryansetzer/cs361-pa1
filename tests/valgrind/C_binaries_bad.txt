==59366== Memcheck, a memory error detector
==59366== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==59366== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==59366== Command: ../dukesh -b scripts/bins_bad.txt
==59366== 
==59366== 
==59366== HEAP SUMMARY:
==59366==     in use at exit: 0 bytes in 0 blocks
==59366==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==59366== 
==59366== All heap blocks were freed -- no leaks are possible
==59366== 
==59366== For lists of detected and suppressed errors, rerun with: -s
==59366== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
