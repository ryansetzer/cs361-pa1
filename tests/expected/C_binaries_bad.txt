./bin/ls: invalid option -- 'l'
./bin/head: invalid option -- 'c'
$ ./bin/ls -l
$ ./bin/head -c 5 Makefile
$ quit

